src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -530,6 +530,8 @@ static void viewport_interaction_remove_footpath(TileElement* tileElement, int32
     tileElement2 = map_get_first_element_at(x / 32, y / 32);
     do
     {
+        if (tileElement2 == nullptr)
```

Comment:
You can just return before going into the `do` function, like you already do elsewhere in the PR.

Reply:
Done!